service/chain/disk.go

```diff
@@ -185,6 +185,42 @@ func (d *Disk) Results(height uint64) ([]*flow.TransactionResult, error) {
 	return converted, nil
 }
 
+func (d *Disk) Seals(height uint64) ([]*flow.Seal, error) {
+
+	blockID, err := d.block(height)
+	if err != nil {
+		return nil, fmt.Errorf("could not get block for height: %w", err)
+	}
+
+	// LookupPayloadSeals() returns the IDs of all the seals in the specified block.
+	// It should not be confused with LookupBlockSeal(), which returns the ID of the
+	// *last* payload seal found in the block.
+
+	var sealIDs []flow.Identifier
+	err = d.db.View(operation.LookupPayloadSeals(blockID, &sealIDs))
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve seal IDs: %w", err)
+	}
```

Comment:
```suggestion
	// LookupPayloadSeals() returns the IDs of all the seals in the specified block.
	// It should not be confused with LookupBlockSeal(), which returns the ID of the
	// *last* payload seal found in the block.
	var sealIDs []flow.Identifier
	err = d.db.View(operation.LookupPayloadSeals(blockID, &sealIDs))
	if err != nil {
		return nil, fmt.Errorf("could not retrieve seal IDs: %w", err)
	}
```

In cases like this, the newlines are detrimental to the readability since it's not clear what the comment applies to. A comment or comment block should always be attached to the code it's commenting on and never separated, IMO.